uefi/src/boot.rs

```diff
@@ -724,6 +724,32 @@ pub unsafe fn exit(
     )
 }
 
+/// Adds, updates, or removes a configuration table entry
+/// from the EFI System Table.
```

Comment:
nit: I thought we want to consistently use UEFI rather than EFI.

We can do that in a tree-wide follow-up soon.

Reply:
Oops, agree. I copied from the method docstring and didn't proofread carefully :)